include/wx/fontutil.h

```diff
@@ -161,8 +161,16 @@ class WXDLLIMPEXP_CORE wxNativeFontInfo
 
     void Free();
 
+    // Font Names in macOS and iOS:
+    // while GetFaceName from now on returns the unique and never localized PostScript name of the font
+    // there are other types of names that are useful
+    
+    // returns the font family name eg Helvetica Neue
     wxString GetFamilyName() const;
+    // returns the font style name eg Condensed Bold
```

Comment:
It would be nice if the comment explained if this is the same as our weight (i.e. if it's going to be "Bold" for a bold font) or if it's independent of it (as I think it is, but I'm not sure).

Reply:
Yes, it is a bold font also in wx sense, but we don't have a flag for condensed, so it's partially overlapping, an additional problem which I have not yet thought fully about are emulated properties, eg I'm emulating italic and bold if there is no corresponding font...